cmd/pod-scaler/admission.go

```diff
@@ -217,11 +224,32 @@ func useOursIfLarger(allOfOurs, allOfTheirs *corev1.ResourceRequirements, logger
 			if our.Cmp(their) == 1 {
 				logger.Debugf("determined %s %s of %s to be larger than %s configured", field, pair.resource, our.String(), their.String())
 				(*pair.theirs)[field] = our
+				recordPodAdmitted(containerName, admittedPodsMetric)
```

Comment:
can this move out of this loop so that it only happens once per pod rather than per metric? that also means it will happen regardless of our determined amount being larger (which is a good thing).